arcgis-ios-sdk-samples/Edit data/Edit features (connected)/EditFeaturesOnlineViewController.swift

```diff
@@ -50,6 +50,13 @@ class EditFeaturesOnlineViewController: UIViewController, AGSGeoViewTouchDelegat
                 
         //hide the sketchToolbar initially
         self.sketchToolbar.isHidden = true
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(EditFeaturesOnlineViewController.sketchChanged(_:)), name: .AGSSketchEditorGeometryDidChange, object: nil)
```

Comment:
FYI: Most of the time notification actions in view controllers are updating the view. When that's the case, it make more sense to tie the lifetime of the notification to the lifetime of the visibility of the view. Register for it in `viewWillAppear` and unregister for the same one in `viewDidDisappear`.